style.css

```diff
@@ -0,0 +1,207 @@
+html, body {
+	margin: 0;
+	padding: 0;
+}
+
+p {
+	margin: 0; 
+	padding: 0; 
+}
+
+.clearfix::after {
+  content: "";
+  display: table;
+  clear: both;
+}
+
+.wrapper {
+	width: 800px;
+	height: 600px;
+	position: absolute;
+	margin: auto;
+	top: 0;
+	right: 0;
+	bottom: 0;
+	left: 0;
+	background: url("src/img/background_image_1.png") no-repeat;
+}
+
+.container {
+	width: 581px;
+	height: 438px;
+	position: absolute;
+	margin: auto;
+	top: 0;
+	right: 0;
+	bottom: 0;
+	left: 0;
+	background: url("src/img/background_photo_1.png") no-repeat;
+	box-shadow: 0px 2px 21px 0px rgba(0, 0, 0, 0.44);
+	font-family: "Dosis", sans-serif;
+}
+
+.column-1,
+.column-2 {
+	width: 289px;
+	min-height: 438px;
+}
+
+.column-1 {
+	float: left;
+	border-right: 1px solid #771921;
+}
+
+.column-1-block-1,
+.column-1-block-2,
+.column-1-block-3 {
+	float: left;
+	width: 289px;
+}
+
+.column-1-block-1 {
+	min-height: 252px;
+	position: relative;
+	overflow: hidden;
+}
+
+.column-1-block-1 img {
+	position: absolute;
+	top: 50%;
+ left: 50%;
+ margin: -25% 0 0 -25%;
+}
+
+.column-1-block-1:before {
+	content: '';
+	position: absolute;
+	top: 0;
+	left: 0;
+	width: 330px;
+	height: 252px;
+	background: linear-gradient(to top, rgba(255,255,255, 0), rgba(255,255,255, 0.45));
+	transform: translateX(-10.6em) skewX(-50deg);
+}
+
+.column-1-block-2,
+.column-1-block-3 {
+	min-height: 93px;
+	opacity: 0.89;
+ color: #ffffff;
```

Comment:
Add spacec to styles. It shouldn't be on the same level as class. Check for other similar cases.